presto-main/src/main/java/com/facebook/presto/sql/planner/plan/WindowNode.java

```diff
@@ -90,6 +90,11 @@ public WindowNode(
         return ImmutableList.copyOf(concat(source.getOutputSymbols(), windowFunctions.keySet()));
     }
 
+    public Set<Symbol> getCreatedSymbols()
```

Comment:
`getCreatedSymbols` sounds like a generic (like every `PlanNode` could have such) name while it is not a generic method as it is specified for `WindowNode`. 

I would name it as `getWindowSymbols`